eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -11,6 +11,10 @@ function CreateReleases($pkgList, $releaseApiUrl, $releaseSha) {
     if ($pkgInfo.ReleaseNotes -ne $null) {
       $releaseNotes = $pkgInfo.ReleaseNotes
     }
+    # As github api limit the body param length with 125000 characters, we have to truncate the release note if needed.
```

Comment:
Can you please provide a link to github documentation about this?

Reply:
I cannot found the exact doc about this. But from the error msg returned, it is clear there is a limitation for it.
![image](https://user-images.githubusercontent.com/1726438/149489447-e45138c9-f560-46da-8984-3ace76d6423f.png)
There is a same issue here: https://github.com/goreleaser/goreleaser/issues/2647